docs/source/algorithms/Q1DWeighted-v1.rst

```diff
@@ -56,8 +56,33 @@ In the reduction workflows this option is deprecated.
 Wedges
 ######
 
-For unisotropic scatterer, the *I(Q)* can be calculated for different angular sectors (or wedges).
-This can be done in two ways: symmetric wedges (default) or asymmetric wedges.
+For anisotropic scatterer, the *I(Q)* can be calculated for different angular sectors (or wedges).
+
+These sectors can be defined in two different ways : either by drawing them, or by defining them.
+There are also two integration mode: symmetric wedges (default) or asymmetric wedges.
+
+Defining wedges
+~~~~~~~~~~~~~~~
+
+Wedges are defined by the ``NumberOfWedges``, ``WedgeAngle`` and ``WedgeOffset`` parameters.
+The trigonometrical circle is split in ``NumberOfWedges`` sectors, equally spaced, spanning a range of ``WedgeAngle``.
+
+Drawing sectors
+~~~~~~~~~~~~~~~
+
+It is also possible to use the instrument viewer to draw the shape of the angular sector. Only sector shapes are currently supported,
+and they must be drawn in the Full 3D, Z- projection, without any rotation (translation and zoom are supported). Once the shapes are
```

Comment:
I think we need to mention here, that in Z-, the X points to the left, unlike in the picture here when there is no shape table. When doing the wedges without the table, trigonometric azimuthal angle opens at positive ray of the x-axis.